ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -18,6 +19,87 @@ import {
   resolvePendingApproval,
 } from '../../../../../store/actions';
 import { confirmSelector } from '../../../selectors';
+import useAlerts from '../../../../../hooks/useAlerts';
+import { MultipleAlertModal } from '../../../../../components/app/confirmations/alerts/multiple-alert-modal';
+import {
+  BackgroundColor,
+  Severity,
+} from '../../../../../helpers/constants/design-system';
+import { getSeverityBackground } from '../../../../../components/app/confirmations/alerts/alert-utils';
+import { Alert } from '../../../../../ducks/confirm-alerts/confirm-alerts';
+
+function ReviewAlertButton({
+  backgroundColor,
+  setAlertModalVisible,
+}: {
+  backgroundColor: BackgroundColor;
+  setAlertModalVisible: (visible: boolean) => void;
+}) {
+  const t = useI18nContext();
+  return (
+    <Button
+      block
+      backgroundColor={backgroundColor}
+      onClick={() => setAlertModalVisible(true)}
+      startIconName={IconName.SecuritySearch}
+      size={ButtonSize.Lg}
+      data-testid="review-alert-button"
+    >
+      {t('reviewAlerts')}
+    </Button>
+  );
+}
+
+function ConfirmFooterButton({
+  hasUnconfirmedAlerts,
+  unconfirmedAlerts,
+  setAlertModalVisible,
+  onSubmit,
+  disabled,
+}: {
+  hasUnconfirmedAlerts: boolean;
+  unconfirmedAlerts: Alert[];
+  setAlertModalVisible: (visible: boolean) => void;
+  onSubmit: () => void;
+  disabled: boolean;
+}) {
+  const t = useI18nContext();
+
+  function severityStyle() {
+    let severity = Severity.Info;
+    unconfirmedAlerts.forEach((alert) => {
+      if (alert.severity === Severity.Danger) {
+        severity = Severity.Danger;
+      } else if (
+        alert.severity === Severity.Warning &&
+        severity !== Severity.Danger
+      ) {
+        severity = Severity.Warning;
+      }
+    });
+    return getSeverityBackground(severity);
+  }
+
+  return hasUnconfirmedAlerts ? (
+    <ReviewAlertButton
```

Comment:
Since we're already in a `ConfirmButton` component, would it be simpler to render a single button here with dynamic properties where needed? So we can have a dynamic `onClick`, `disabled` and text etc.

Reply:
Sure, I removed the `ReviewAlertButton` and added the dynamic logic to support the properties required by those two cases.